index.html

```diff
@@ -37,9 +87,9 @@
     </div>
     <div class="desktop-header">
       <nav>
-        <a href="#recent-works-start">Portfolio</a>
-        <a href="#about-start">About</a>
-        <a href="#contact-start">Contact</a>
+        <div id="recent-tag" class="link"><a href="#recent-works-start">Portfolio</a><div class="underline"></div></div>

```

Comment:
The project details section on the desktop view has a bit more height on the design page. Kindly see the design and make this section exactly the same as in the design.

Reply:
Hello I'm not sure what you mean by the desktop view has a bit more. height. on my computer it seems to have the correct display.